contents/spotlight/startup-before-sunset-ai.md

```diff
@@ -0,0 +1,71 @@
+
+### 2. Prioritize community and creativity
+
+Focusing on community engagement is an important part of any launch plan, according to Zekiye — and the earlier you start, the more successful you’ll be. 
+
+“We chose to focus on the Product Hunt community and we knew that would require us to be active on the platform,” she explains. “That means finding ways to add value; ways you can support them whenever you’re in touch so that, in turn, you can benefit from the community.”
+
+Creative assets were also a major focus for BeforeSunset AI. The team created new product screenshots and videos ahead of the launch, and carefully considered how to make the launch page look as on-brand as possible even as the product continued to change and improve internally.
+
+Finding the right community or launch channel can be a significant challenge all by itself and requires a keen understanding of your target audience. Knowing who users are, what their pain points are, and where they look for solutions is just as important to selecting a launch channel as experimentation.
+
+"We're always working on better defining our target audience," says Zekiye. "Nailing the persona is especially hard, because we often see users interacting with the product in ways we don't expect."
+
+One example of this came from BeforeSunset's 'Plan my day' tool, where users are intended to add tasks and let the AI schedule time for them. [Session replays](/session-replay) showed users were instead adding their own timeblocks in the description, which helped the team understand how busy their users were and what tasks they spent their time on.
+
+> **Putting it into action:** Document your target audiences and personas somewhere that everyone can access, then use this to identify the communities where these users may congregate. Use the persona as a benchmark for the quality of your marketing assets, and be sure to engage with communities well in advance of a launch to build goodwill. 
+
+### 3. Create an outreach strategy
+
+With Product Hunt, you only have a short amount of time in order to get traction and galvanize the community. Take too long and you’ll fall off the daily page, and into the archives. So, you need a strategy to continue currying favor and votes throughout the day.
+
+“We created a spreadsheet of our users and ran some one-on-one outreach to them,” says Zekiye Nur Kesici. “We even had a competition in the marketing team, offering a prize to whoever could reach out to the most people via LinkedIn. We also ran a webinar, published a blog post, used social media, and leveraged our waitlist to collect votes.”
```

Comment:
Random thought... could they provide a screenshot / example of one of their outreach messages. Would really lift this if we could say here's how to write a good message asking for a vote.